test/e2e/alertmanager_test.go

```diff
@@ -509,3 +510,48 @@ func TestAlertmanagerDisabling(t *testing.T) {
 		}
 	})
 }
+
+func TestAlertManagerHasAdditionalAlertRelabelConfigs(t *testing.T) {
+	const (
+		expectPlatformLabel      = "openshift_io_alert_source"
+		expectPlatformLabelValue = "platform"
+	)
+
+	var alerts alerts
+
+	err := framework.Poll(5*time.Second, time.Minute, func() error {
+		resp, err := f.AlertmanagerClient.Do("GET", "/api/v2/alerts", nil)
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+
+		if resp.StatusCode != http.StatusOK {
+			return fmt.Errorf("expecting 200 status code, got %d (%q)", resp.StatusCode, resp.Body)
+		}
+
+		if err := json.NewDecoder(resp.Body).Decode(&alerts); err != nil {
+			return fmt.Errorf("error decoding alert response")
+		}
+
+		return nil
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, alert := range alerts {
+		v, found := alert.Labels[expectPlatformLabel]
+		if !found {
+			t.Fatal("expected correct label to be present")
+		}
+
+		if v != expectPlatformLabelValue {
+			t.Fatalf("expected correct value for %s but got %s", expectPlatformLabel, v)
+		}
+	}
+}
+
+type alerts []struct {
```

Comment:
It may be better to move this type to line 520 as an inline declaration if we are not using it for other tests?